src/index.js

```diff
@@ -226,6 +226,45 @@ export default async function microbundle(options) {
 	);
 }
 
+function createBabelConfig(options) {
+	const partialConfig = loadPartialConfig();
+	const hasBabelConfigFile = partialConfig.hasFilesystemConfig();
```

Comment:
Note that this function does not respect the `babel` field in package.json

Reply:
The thing is that `loadPartialConfig` is meant to be called per-file, because some files may have a `.babelrc` and some may not. That is what drove my comment above. It seems like `rollup-plugin-babel` should be exposing this as a feature, which is what we do for `babel-loader` so that people can do the same thing.

Reply:
@loganfsmyth: I see, thanks for the clarification. So if we go for that solution, that would mean this is impeded on `rollup-plugin-babel` exposing this feature.

I can make an issue over at rollup-plugin-babel to the best of my ability and see where that takes us.